src/sentry/templates/sentry/emails/confirm_email.txt

```diff
@@ -0,0 +1,7 @@
+Thanks for signing up for Sentry! Please confirm your email ({{ user.username|safe }}) by clicking the link below:
```

Comment:
Wouldn't this email get sent for other cases instead of signing up? Like, adding a new email address or changing an existing.